Source/Blazorise/Link.razor.cs

```diff
@@ -88,6 +88,7 @@ protected override void OnParametersSet()
 
             absoluteUri = To == null ? string.Empty : NavigationManager.ToAbsoluteUri( To ).AbsoluteUri;
             active = ShouldMatch( NavigationManager.Uri );
+            DirtyClasses();
```

Comment:
@stsrki I had to add this line to get the `active` class appearing on the `Link` when its within the `BarDropdownItem` in AntDesgin (and only for AntDesgin!)  I cannot figure out why this happening.. 🤔
Appears to be some sort of race condition with this method (`OnParametersSet`) and `OnLocationChanged`?
Seems like this may cause performance issues.. Maybe you could take a quick look, if you get a chance?

Reply:
Maybe we can override Link for AntDesign? I can try it later.